README.md

```diff
@@ -69,7 +69,7 @@ Make a directory for building
 Run cmake and make
 
     $ cmake ../root
-    $ make -j8
+    $ make -jn # where n is the number of cores 
```

Comment:
```suggestion
    $ make -j n # where n is the number of cores 
```
is easier to read - what do you think?

Reply:
Yes, it is easier to read, however is it possible that the reader might add the space between the j and the number?